spec/features/play_spec.rb

```diff
@@ -0,0 +1,16 @@
+
+feature 'Enter names' do
```

Comment:
It looks like your feature test may be about the RPS options. Should this be renamed something like 'RPS options' rather than 'Enter names'?